printer_mods/ppak10/Cable_Port_Flush_Mount_Skirt/README.md

```diff
@@ -0,0 +1,38 @@
+# Cable Port Flush Mount Skirt
+
+Provides an inlet to mount BATIGE circular cable adapter ports on the rear
+center skirt (250, 300, and 350) allowing easy access to display and USB headers
+of the raspberry pi.
+
+It can be used for quick access to Raspberry Pi ports often out of reach
+underneath the printer without needing to flip over the printer.
+
+<div style="justify-content: center; display: flex;">
+  <img src="./Images/250_300_350_Render.png" alt="CAD" width="750"/>
+</div>
+
+<div style="justify-content: center; display: flex;">
+  <img src="./Images/250.jpeg" alt="250" width="250"/>
```

Comment:
this image, and all of the other .jpeg's here appear as broken image links, due to the links containing `.jpeg` while the files are actually named `.JPEG`
![image](https://github.com/VoronDesign/VoronUsers/assets/105010153/aa59875b-e769-4158-8487-3827fa98084d)